### PR TITLE
add r-anomalydetection r-bradleyterry2  r-broom  r-forecast  r-htmltable  r-microbenchmark  r-mlmrev  r-multcomp

### PR DESCRIPTION
**Pull request by bgruening**
xref: https://github.com/conda-forge/staged-recipes/issues/2009
ping @johanneskoester 

r-anomalydetection
 r-bradleyterry2
 r-broom
 r-forecast
 r-htmltable
 r-microbenchmark
 r-mlmrev
 r-multcomp

---------------------
adds

r-car
r-caret
r-gmp
r-matrixstats
r-qdaptools
r-tidyverse

### CODE REVIEW

File: recipes/r-gmp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+    - gmp >=5.0.1,<7
```
Comment (isuruf) [inline recipes/r-gmp/meta.yaml]: Change this to `- {{native}}gmp            # [win]` as in `r-base`

File: recipes/r-gmp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  run:
+    - r-base
+    - libgcc  # [not win]
+    - gmp >=5.0.1,<7
```
Comment (isuruf) [inline recipes/r-gmp/meta.yaml]: Same here

File: recipes/r-gmp/meta.yaml (hunk 1 above)
Comment (bgruening) [inline recipes/r-gmp/meta.yaml]: But this will only use gmp for windows builds, we also need it for Linux, isn't it?

Comment (isuruf) [inline recipes/r-gmp/meta.yaml]: Yes, sorry about that. Do `- {{native}}gmp >=5.0.1,<7`

Comment (bgruening) [inline recipes/r-gmp/meta.yaml]: Done!

**Comment by isuruf** [comment]
`r-pbkrtest` is in the other PR

File: recipes/r-anomalydetection/meta.yaml [hunk 3]
```diff
@@ -16,6 +16,7 @@ source:
 
 build:
   number: 0
+  # r-tidyverse is not available under windows
   skip: true  # [win32]
```
Comment (isuruf) [inline recipes/r-anomalydetection/meta.yaml]: You need to change this

**Comment by isuruf** [comment]
Now that `r-psych` for win is in, do you want to enable `r-broom`, `r-modelr` and `r-tidyverse`?

**Comment by bgruening** [comment]
I guess this is the cleanest way, but it will take an other 2h to test :)

**Comment by isuruf** [comment]
You can always do it in the feedstock.

**Comment by bgruening** [comment]
Ok, then let's do it in the feedstocks. This way I practice it a little bit and I can finish up level6 today maybe.

**Comment by bgruening** [comment]
The win error was because of failing downloads and 2 out of 3 were passing.

**Pull request merged**